plugins/performance-lab/tests/includes/site-health/audit-autoloaded-options/test-audit-autoloaded-options.php

```diff
@@ -177,14 +172,43 @@ static function () {
 		// Test that the reverted autoloaded option is displayed in the autoloaded options perflab_aao_get_autoloaded_options_table().
 		$table_html = perflab_aao_get_autoloaded_options_table();
 		$this->assertStringContainsString( self::AUTOLOADED_OPTION_KEY, $table_html );
+	}
 
-		// Remove the mock filter.
-		remove_filter(
-			'wp_redirect',
-			static function () {
-				return false;
-			}
-		);
+	public function test_perflab_aao_autoloaded_options_auto_enable_functionality(): void {
```

Comment:
Would be good to add the `@covers` tags for which functions this is intending to test.